Create ui-c8y-1020-14-10-changes-to-unit-in-the-dp-template-is-now-reflected-in.md (#2346)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-14-10-changes-to-unit-in-the-dp-template-is-now-reflected-in.md

---
date: ""

> title: Changes to the unit in the data point template is now reflected in the info gauge widget view.

Reviewer: ```suggestion
title: Changing the unit in a data point template is now correctly reflected in related "Info gauge" widgets
```

> title: Changes to unit in the dp template is now reflected in the info gauge widget view. (#6823) [GRAFT][release/cd] (#6838)

Reviewer 2: ```suggestion
title: Changes to the unit in the data point template is now reflected in the info gauge widget view.
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60261
version: 1020.14.10
---

> Changes to unit in the dp template is now reflected in the info gauge widget view. (#6823) [GRAFT][release/cd] (#6838)

Reviewer 2: ```suggestion
In the past, when changing the unit of a data point template, the change was not reflected in the info gauge widget which was using this data point. This inconsistent behavior has now been fixed. Now, whenever the unit is updated in a data point template, this change will also be immediately visible in the info gauge widget view.
```

> In the past, when changing the unit of a data point template, the change was not reflected in the info gauge widget which was using this data point. This inconsistent behavior has now been fixed. Now, whenever the unit is updated in a data point template, this change will also be immediately visible in the info gauge widget view.

Reviewer: ```suggestion
In the past, when changing the unit of a data point template, the change was not reflected in the "Info gauge" widgets which were using this data point. This inconsistent behavior has now been fixed. Now, whenever the unit is updated in a data point template, this change will also be immediately visible in the respective "Info gauge" widgets.
```
